CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ We use [semantic versioning](http://semver.org/):
 - [breaking change] removed options `coverage-via-http` and `teamscale-testwise-upload`. Use `tia-mode=http` or `tia-mode=teamscale-upload` instead.
 - [breaking change] _tia-client_: changed Java API
 - [feature] always exclude common libraries from profiling to shrink coverage files
+- [feature] enable changing and reading the commit message via http
```

Comment:
"via http" is a bit imprecise. (HTTPS, too?) Better "via a new `/message` REST endpoint"